### PR TITLE
change bracket notation to dot notation

### LINKED ISSUE #329: Code review: Dot notation instead of bracket
Refs #274 / https://github.com/DoSomething/LetsDoThis-iOS/blob/694848458f8930dddc3c75c7a36e36dc3d962273/Lets%20Do%20This/Controllers/Campaign/LDTCampaignDetailViewController.m#L110

My n00b-ness has definitely mixed up dot syntax and bracket all over place.  I didn't realize `uppercaseString` can also be a property of a `NSString` -- I've always been using `[someString uppercaseString]`.

We should change all instances to `someString.uppercaseString` to keep things cleaner / consistent. There are plenty of other places where this is happening too :person_frowning:

### PR DESCRIPTION
**Pull request by tongxiang**
#### What's this PR do?
Specifically changes `[xxxx upperCaseString]` to `xxxx.upperCaseString` and `[xxxx count]` to `xxxx.count`

#### What are the relevant tickets?
Closes #329. 

#### Questions:
@aaronschachter any other instances you can think of that need to be changed?

### CODE REVIEW

File: Lets Do This/Controllers/Profile/LDTUserProfileViewController.m [hunk 1]
```diff
@@ -115,7 +115,7 @@ - (void)styleView {
     self.headerView.backgroundColor = [UIColor colorWithPatternImage:[UIImage imageNamed:@"Header Background"]];
     [self styleBackBarButton];
 
-    self.nameLabel.text = [self.nameLabel.text uppercaseString];
+    self.nameLabel.text = self.nameLabel.text.uppercaseString;
     [self.nameLabel setFont:[LDTTheme fontTitle]];
```
Comment (aaronschachter) [inline Lets Do This/Controllers/Profile/LDTUserProfileViewController.m]: `self.nameLabel.font = [LDTTheme fontTitle];`

File: Lets Do This/Controllers/Profile/LDTUserProfileViewController.m [hunk 2]
```diff
@@ -115,7 +115,7 @@ - (void)styleView {
...
     [self.nameLabel setTextColor:[UIColor whiteColor]];
```
Comment (aaronschachter) [inline Lets Do This/Controllers/Profile/LDTUserProfileViewController.m]: `self.nameLabel.textColor = [UIColor whiteColor];`

**Comment by aaronschachter** [comment]
Found two -- might find similar in the `styleView` methods.

**Comment by tongxiang** [comment]
Changed `LDTTheme` and `UIColor` bracket notation to dot notation--I think this may be complete @aaronschachter.

**Comment by aaronschachter** [comment]
Sure looks it! Awesome job! :+1:

File: Lets Do This/Controllers/Profile/LDTUserProfileViewController.m [hunk 3]
```diff
@@ -115,18 +115,18 @@ - (void)styleView {
     self.headerView.backgroundColor = [UIColor colorWithPatternImage:[UIImage imageNamed:@"Header Background"]];
     [self styleBackBarButton];
 
-    self.nameLabel.text = [self.nameLabel.text uppercaseString];
-    [self.nameLabel setFont:[LDTTheme fontTitle]];
-    [self.nameLabel setTextColor:[UIColor whiteColor]];
+    self.nameLabel.text = self.nameLabel.text.uppercaseString;
+    [self.nameLabel setFont:LDTTheme.fontTitle];
```
Comment (aaronschachter) [inline Lets Do This/Controllers/Profile/LDTUserProfileViewController.m]: Hmm theres still more here (of course I catch after I merge. `self.nameLabel.font`

Comment (aaronschachter) [inline Lets Do This/Controllers/Profile/LDTUserProfileViewController.m]: I'll fix the ones in `LDTUserProfileViewController` -- I'm in that file now and just rebased with your changes.

**Pull request merged**